hypersync-client/src/column_mapping.rs

```diff
@@ -311,19 +358,24 @@ mod tests {
 
     #[test]
     fn test_signed_binary_to_target() {
-        const RAW_INPUT: &[i64] = &[-69, 0, 69, -1, 1];
+        const RAW_INPUT: &[i64] = &[-69, 0, 69, -1, 1, i64::MAX, i64::MIN];
```

Comment:
I guess it could be beneficial to test with even larger numbers than i64?

Reply:
yeah probably a good idea to add another test for that :+1: